src/compiler/moduleNameResolver.ts

```diff
@@ -276,7 +276,7 @@ namespace ts {
     }
     const nodeModulesAtTypes = combinePaths("node_modules", "@types");
 
-    function getPnpTypeRoots(currentDirectory: string) {
+    export function getPnpTypeRoots(currentDirectory: string) {

```

Comment:
iirc when new symbols are exported things become more annoying PR-wise as there are extra files to check-in (it's why I avoided reusing `isPnpAvailable` across files).